### DISCUSSION TITLE
Should I report bugs while on the nightly channel?

### DISCUSSION THREAD

**Discussion by TCROC** [consulo/consulo#437] [Q&A] [answered]
I'm currently on the nightly channel in order to use the UPM Git package feature for Unity.  Sometimes I encounter bugs on this channel in other parts of the Consulo.  Should I be reporting these bugs?  Or should I only report bugs from the other channels?

**Comment by VISTALL** [comment, marked as answer]
Hello. Yes you should. Release(and other channels) is point inside nightly dev line. Since often I'm only report issues - will be nice have other issue reporters.

**Reply by TCROC** [reply]
Awesome!  I can help out with that! :)
